android/app/build.gradle

```diff
@@ -165,6 +166,15 @@ android {
             targetSdkVersion 29
             resValue "string", "build_config_package", "com.emurgo"
         }
+        nightly {
+            dimension "version"
+            applicationIdSuffix ".nightly"
+            applicationId 'com.emurgo'
```

Comment:
Needs a different application ID to be uploaded as a separate app in the play store

Reply:
the resulting applicationId should be `com.emurgo.nightly`